example-app/src/main/java/com/adyen/checkout/example/ui/theme/ExampleTheme.kt

```diff
@@ -0,0 +1,120 @@
+package com.adyen.checkout.example.ui.theme
```

Comment:
Some of these new files are missing copyrights

Reply:
Oh no not again

Reply:
Fixed it!